examples/hosts/app-integration/external-data/src/view/recoilState.ts

```diff
@@ -0,0 +1,11 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { atom } from "recoil";
+
+export const localUnsavedChangesState = atom({
+	key: "localUnsavedChanges",
+	default: 0,
```

Comment:
Nit: It would be nice if we could initialize this based on the model's unsync'd change count at the time, since technically it could be non-0 (e.g. a collaborator joining the session after some local changes were made, but before they were synced). Not sure how easy this is to do in recoil

Reply:
Removed this entirely since it's unneeded.